lib/lighthouse/facilities/client.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'lighthouse/facilities/response'
+
+module Lighthouse
+  module Facilities
+    # Documentation located at:
+    # https://developer.va.gov/explore/facilities/docs/facilities
+    class Client < Common::Client::Base
+      attr_accessor :headers
+
+      configuration Lighthouse::Facilities::Configuration
+
+      def initialize(api_key)
```

Comment:
this needs to call super to pick up any additional initializations in the base class.

Reply:
is this just to make testing easier?